.github/workflows/deploy.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
 
       - uses: actions/setup-node@v3
         with:
-          node-version: 14.17.5
+          node-version: 16.16.0
```

Comment:
Love this solution. I wonder if it would make sense to read the version from the `.node-version` file rather than setting it here manually. @rpdelaney this is another good chance to reduce toil here.

Reply:
Yeah, that would make sense to me. I can look into it after we merge this PR.

Reply:
As far as I know, awareness of `.node-version` would have to be built-in to the `setup-node` action. From the [README](https://github.com/actions/setup-node#usage):
> The node-version input is optional. If not supplied, the node version from PATH will be used. However, it is recommended to always specify Node.js version and don't rely on the system one.
It sounds like we could remove this and it might do what we want, but that's discouraged. I don't know why this is discouraged.